feature-swap-impl/src/main/java/io/novafoundation/nova/feature_swap_impl/domain/interactor/SwapInteractor.kt

```diff
@@ -139,6 +139,8 @@ class SwapInteractor(
 
             positiveAmountOut()
 
+            sufficientBalanceInFeeAsset()
```

Comment:
changed validation index since it should stay before checking feeAsset.transferable > fee + ed